nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/ErrorHandler.scala

```diff
@@ -22,7 +22,7 @@ object ErrorHandler {
       }
     } finally {
       errors.close()
-      fileSystem.close()
+//      fileSystem.close()
```

Comment:
great change！
please check the scalafmt style. Maybe you can remove this line.

Reply:
It has been deleted.